src/main/java/com/gmail/goosius/siegewar/events/SiegeRemoveEvent.java

```diff
@@ -86,4 +91,12 @@ public String getBesiegedTownName() {
     public String getDefenderName() {
         return defenderName;
     }
+
+    public Nation getNation() {
+        return nation;
+    }
+
+    public String getMessage() {
+        return LegacyComponentSerializer.legacySection().deserialize(Translatable.of("msg_swa_remove_siege_success").defaultLocale() + " " + getDefenderName()).content();
```

Comment:
This is going to broadcast a somewhat janky-looking String: 

`Successfully removed siege. SiegedTownName`

It should probably get a new language string. Just add it once to the en-US.yml and crowdin will make it show up in the other language files.